apis/v1alpha1/forwarding_target_types.go

```diff
@@ -62,9 +64,8 @@ type HTTPForwardToTarget struct {
 	// Support: Core (HTTPRoute)
 	// Support: Extended (TCPRoute)
 	//
-	// +optional
 	// +kubebuilder:default=1
-	Weight TargetWeight `json:"weight"`
+	Weight int32 `json:"weight,omitempty"`
```

Comment:
Min: 1
Max: 10000

Reply:
Question: Is it worth defining a `Port` type with the min, max and default and then use that everywhere?
I understand that you still need to specify the kubebuilder annotation in the place where it is used but having a type would be sort of a self-documenting thing through the API.